tests/cases/compiler/jsFileFunctionParametersAsOptional2.ts

```diff
@@ -0,0 +1,18 @@
+// @allowJs: true
+// @noEmit: true
+
+// @filename: foo.js
+/**
+ * @param a
+ * @param b
+ * @param c

```

Comment:
just changed that based on @DanielRosenwasser's comments, so now it says if it has **any** parameter with jsdoc comments it is not considered untyped.